schema/argus.schema

```diff
@@ -10,18 +10,18 @@ type argus.record = record{
   // Standard fields that are always present.
   StartTime: time #timestamp,     // stime
   Flgs: string,                   // flgs
-  Proto: string #index=hash,      // proto
+  Proto: string #id,              // proto
```

Comment:
`:%s/#id,/`

That was a placeholder attribute. We can just revert this commit again when it's fixed, so we don't really need the placeholder.

Reply:
I'm not sure about this one, I think it's less surprising if a revert is **just** a revert, rather than having revert + fix-ups in the same commit. I can of course remove the `#id` in a separate commit, but then we couldn't cleanly re-apply the original commit after we fixed the underlying performance issues. So as long as the `#id` attribute does no harm, I'd prefer just keeping it as-is.

Reply:
Fine by me as well.